src/test/java/org/dependencytrack/policy/cel/mapping/FieldMappingUtilTest.java

```diff
@@ -58,7 +58,7 @@ private void assertValidProtoFieldsAndColumns(final Class<?> projectionClazz,
                     assertHasProtoField(protoDescriptor, fieldMapping.protoFieldName());
                     //skipping the published_at column for sql check because functionality wise the Component model
                     // class does not need the published_at column from integrity_meta and this is breaking the unit test
-                    if (fieldMapping.sqlColumnName().equals("PUBLISHED_AT")) {
+                    if (fieldMapping.sqlColumnName().equals("PUBLISHED_AT") || fieldMapping.sqlColumnName().equals("LATEST_VERSION")) {
```

Comment:
These checks can be avoided when removing the `@MappedField` annotations from the respective fields in the projection.